images/linux/scripts/installers/docker-moby.sh

```diff
@@ -50,3 +57,8 @@ docker pull ubuntu:14.04
 echo "Documenting Docker version"
 docker_version=$(docker -v)
 DocumentInstalledItem "Docker-Moby ($docker_version)"
+
+echo "Documenting Docker-buildx version"
+DOCKER_BUILDX_VERSION=$(apt-cache policy moby-buildx | grep Installed)
```

Comment:
Why do we use so complex syntax?
`docker buildx version`

Reply:
```
testAdm2@danurgDocker:~$ command -v docker
/usr/bin/docker

testAdm2@danurgDocker:~$ command -v docker buidx
/usr/bin/docker

testAdm2@danurgDocker:~$ command -v docker buidxxx
/usr/bin/docker
```

command -v doesn't work for docker plugins

Reply:
```
testAdm2@danurgDocker:~$ docker buildx version

github.com/docker/buildx 0.3.1+azure 6db68d029599c6710a32aa7adcba8e5a344795a7
```
this is more hard for parsing then `apt-cache policy moby-buildx | grep Installed` result

Reply:
Really `docker buildx version | cut -d ' ' -f2` ?

Reply:
why didn't you immediately provide your best approach without long conversation ?